### ISSUE TITLE
flag for cast to any not rendered types

### ISSUE THREAD

**Issue by francescoagati** [nadako/hxtsdgen#17]
hi,
in some case (enum abstract) hxtsdgen block the output process with this error. 
i think that should be better launch a warning but without block the process and cast to any. 
or have a flag for don't throw an exception and only  a warning and cast the unknown type to any 

https://github.com/nadako/hxtsdgen/blob/bfb4062eb6ab519e8091aded83fcdea123fcf792/src/hxtsdgen/TypeRenderer.hx#L85

**Comment by elsassph** [comment]
What branch are you testing? Is that `master` or branch `feature/6_include-related` in PR?

**Comment by elsassph** [comment]
But I agree a warning would be better

**Comment by francescoagati** [comment]
master

**Comment by elsassph** [comment]
Please try the branch if you don't mind

**Comment by cancerberoSgx** [comment]
Is there a PR for this ? If fixed please could you update haxelib repo ? I currently need to comment that line - just a warning is enough here since probably will take some time to support all clases... Should I PR ? Maybe a flag as argument / macro to throw. (by default should not) Thanks, nice project!

**Comment by elsassph** [comment]
You're right, it shouldn't throw but print a warning. Please PR if you can.
